src/Command/PopulateCommand.php

```diff
@@ -185,7 +185,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
     private function populateIndex(OutputInterface $output, $index, $reset, $options)
     {
         $event = new IndexPopulateEvent($index, $reset, $options);
-        $this->dispatcher->dispatch(IndexPopulateEvent::PRE_INDEX_POPULATE, $event);
+        $this->dispatcher->dispatch($event, IndexPopulateEvent::PRE_INDEX_POPULATE);
```

Comment:
This won't work on < Symfony 4.3

Reply:
So how is this handled then?  A new version of the bundle?  Symfony 4.4 handles it by checking the argument order and calling a legacy handler (and throwing a deprecation warning).  Obviously, we don't want to not support Symfony 5 in favor of supporting older versions of Symfony, so how is this handled? (I don't have much experience with supporting bundles, even branches, tagging and pull requests are new territory for me).

Reply:
Um, so what's a better solution for supporting Symfony 5?  A LegacyDispatcher class?  How do you handle Contracts?  Is it something you can do, as this is a key bundle for some of us.  Thanks.

Reply:
I would check if `$dispatcher` implements `Symfony\Contracts\EventDispatcher\EventDispatcherInterface` [as suggested here](https://github.com/liip/LiipImagineBundle/pull/1212#issuecomment-523840038).